packages/forms/src/components/Fields/Fields.components.tsx

```diff
@@ -95,7 +95,7 @@ export const Fields: React.FC<FieldsProps> = React.memo(props => {
                                         <TextField
                                             {...({
                                                 ...commonProps,
-                                                value: value || '',
+                                                value: value == null ? '' : value,
```

Comment:
We can use [Nullish Coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) operator over here. So You can write like this `value: value ?? ''`. This will handle `undefined` also.